### ISSUE TITLE
Couldn‘t reproduce the results in the paper

### ISSUE THREAD

**Issue by Dopamine0717** [hikvision-research/SSOD#15]
Following readme construction, I only got AP 23.5 after training on Seed 1 Percent 1 Coco standard while 25.81 ± 0.28  is claimed in the paper. Here's my setting:

Dataset:
percent 1 seed 1 Coco standard

Pretrained baseline:
I just changed the number of gpu in the train_gpu2.sh to get the train_gpu4.sh and only got AP 12.3.

Training:
I use train_gpu4.sh with pretrained baseline and got AP 23.5

Config: 
labelmatch_standard.py
samples_per_gpu = 1
total_iter = 250000
update_interval = 1000
test_interval = 2000
learning_rate = 0.005
lr_config = dict(policy='step',warmup='linear',warmup_iters=4000,warmup_ratio=0.001,step=[140000,200000])

Environment:
I used two envs, one with Python = 3.6, mmdet=2.10.0, pytorch=1.6.0, and another with Python = 3.9, mmdet = 2.25, torch = 1.12. Both got the same results. 

Could you share your model weight and log of Label Match training to help use reproduce results?

**Comment by AceCoooool** [comment]
The log file [1-1-40k](https://github.com/hikvision-research/SSOD/blob/13c43fc567ae4c9dfbf74f973e8a5ed20070e984/docs/log/labelmatch_1_1_40k.log) is uploaded. (It's the setting in the ablation study. The final results use longer iterations and stronger resize)
